analyzers/src/SonarAnalyzer.CFG/Sonar/CSharpControlFlowGraph.cs

```diff
@@ -18,18 +18,32 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
 namespace SonarAnalyzer.CFG.Sonar
 {
     public static class CSharpControlFlowGraph
     {
         public static bool TryGet(SyntaxNode node, SemanticModel semanticModel, out IControlFlowGraph cfg)
         {
             cfg = null;
+            var body = node switch
+            {
+                BaseMethodDeclarationSyntax n => (SyntaxNode)n.Body ?? n.ExpressionBody(),
+                PropertyDeclarationSyntax n => n.ExpressionBody?.Expression,
+                IndexerDeclarationSyntax n => n.ExpressionBody?.Expression,
+                AccessorDeclarationSyntax n => (SyntaxNode)n.Body ?? n.ExpressionBody(),
+                AnonymousFunctionExpressionSyntax n => n.Body,
+                ArrowExpressionClauseSyntax n => n,
+                _ when LocalFunctionStatementSyntaxWrapper.IsInstance(node) && (LocalFunctionStatementSyntaxWrapper)node is var local =>
+                    (SyntaxNode)local.Body ?? local.ExpressionBody,
```

Comment:
It's not the prettiest line around, but it fits on a single line
```suggestion
                _ when LocalFunctionStatementSyntaxWrapper.IsInstance(node) && (LocalFunctionStatementSyntaxWrapper)node is var local => (SyntaxNode)local.Body ?? local.ExpressionBody,
```